R/check_enough_data.R

```diff
@@ -0,0 +1,201 @@
+#' Check the dataset contains enough data points.
+#'
+#' `check_enough_data` creates a *specification* of a recipe
+#'  operation that will check if variables contain enough data.
+#'
+#' @param recipe A recipe object. The check will be added to the
+#'  sequence of operations for this recipe.
+#' @param ... One or more selector functions to choose variables for this check.
+#'  See [selections()] for more details. You will usually want to use
+#'  [recipes::all_predictors()] and/or [recipes::all_outcomes()] here.
+#' @param n The minimum number of data points required for training. If this is
```

Comment:
Agree with @dshemetov from elsewhere: can we rename this arg? Perhaps `min_required_n`?

Reply:
... maybe `min_observations`?